packages/dev/core/src/Engines/nativeEngine.ts

```diff
@@ -945,6 +945,10 @@ export class NativeEngine extends Engine {
         this._commandBufferEncoder.finishEncodingCommand();
     }
 
+    public override setStateCullFaceType(_cullBackFaces?: boolean, _force?: boolean): void {

```

Comment:
Not sure if we need a new function or we could do it within the state ? knowing that otherwise it also needs to be in native ?

Reply:
I needed a separate function for the geometry renderer (https://github.com/BabylonJS/Babylon.js/pull/16262/files#diff-fc2462ee78aa57d4d2e939a53a4dda62237a6f11a0f2ca0343df95c3bfce77d6R291-R293), because I must be able to reverse this setting on user request (we need to generate a depth buffer from back faces when we use the automatic thickness computation in SSR).